test/qix-create-reload-doc.spec.js

```diff
@@ -81,39 +101,37 @@ describe('QIX doc sync - reload doc', () => {
       url: `ws://${engine2}/app/engineData/identity/${+new Date()}`,
       createSocket: url => new WebSocket(url),
     });
+    session2.on("traffic:*", (direction, data) => console.log("session2", direction, JSON.stringify(data)));
 
     const qix2 = await session2.open();
-    app = await qix2.openDoc(appId);
-    appLayout = await app.getAppLayout();
-    const reloadTime = appLayout.qLastReloadTime;
-    console.log(`Doc was reloaded last time: ${reloadTime}`);
-    expect(reloadTime).to.equal(firstReloadTime);
-
-    await app.setScript(script);
-    await app.doReload(0, false, false);
-    await app.doSave();
+    const app2 = await qix2.openDoc(appId);
 
-    appLayout = await app.getAppLayout();
-    const newReloadTime = appLayout.qLastReloadTime;
-    console.log(`Reload time after reload: ${newReloadTime}`);
-    expect(newReloadTime).to.not.equal(firstReloadTime);
-    await session2.close();
-    console.log('Closed session to second qix-engine');
+    console.log('Setting a script, reloading doc and saving in second qix-engine instance');
+    await app2.setScript(script);
+    await app2.doReload(0, false, false);
+    await app2.doSave();
 
-    // Open doc in first engine again and verify reload time
-    console.log('Opening a session to first qix-engine and verifying reload time');
-    const session3 = enigma.create({
-      schema,
-      url: `ws://${engine1}/app/engineData/identity/${+new Date()}`,
-      createSocket: url => new WebSocket(url),
-    });
+    // Get and save new reload time
+    const appLayout2 = await app2.getAppLayout();
+    console.log(`Doc after reload has last reload time: ${appLayout2.qLastReloadTime}`);
+
+    // Wait for the first instance to get an updated reload time
+    let newReloadTime = appLayout.qLastReloadTime;
+
+    let retries = 0;
+    while (newReloadTime !== appLayout2.qLastReloadTime && retries < 10) {
+      await sleep(2000);
```

Comment:
Why are we sleeping here?

Reply:
Every other second we check if the `qLastReloadTime` has changed in the first engine session.